sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -14,8 +14,8 @@
         ><template #buttons v-if="!viewOnly">
           <v-row class="p-0 m-0">
             <v-btn
-              color="primary"
-              variant="outlined"
+              color="danger"
+              variant="elevated"
```

Comment:
Between `outlined` and `elevated`, should it not be the same for all the buttons?
Maybe some button can have a "variant" as text, but I was not expecting a specific variant for the cancel button.

Reply:
That's needed for the button to be colored inside:
![image](https://github.com/user-attachments/assets/5db7ac94-d961-43b1-b4ec-f801a4398da5)

Reply:
My point is, for the footer buttons, I do not see us following two configs and if we are it may be a mistake.
My suggestion was to keep as `elevated` for both buttons inside the footer component.